tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -28,15 +28,22 @@ When('the user declines all shares displayed in the notifications on the webUI',
   return client.page.phoenixPage().declineAllSharesInNotification()
 })
 
-Given('app {string} has been enabled', function (app) {
+Given('app {string} has been {}', function (app, action) {
+  assert.ok(
+    action === 'enabled' || action === 'disabled',
+    "only supported either 'enabled' or 'disabled'. Passed: " + action
+  )
   const headers = httpHelper.createAuthHeader('admin')
+  const method = action === 'enabled' ? 'POST' : 'DELETE'
+
   return fetch(client.globals.backend_url + '/ocs/v2.php/cloud/apps/' + app + '?format=json', {
     headers,
-    body: {},
-    method: 'POST'
+    method
   }).then(res => {
-    httpHelper.checkStatus(res, 'Failed while trying to enable the app')
+    httpHelper.checkStatus(res, 'Failed while trying to getting the app' + action)
```

Comment:
what is the error about?
do you want to output `Failed while trying to enable the app` and `Failed while trying to disable the app`?
```suggestion
    httpHelper.checkStatus(res, 'Failed while trying to ' + action + ' the app')
```

similar the next check

Reply:
no, it's whatever we receive as action (though, it can be of either `disabled` or `enabled` in v2/v3 form). So, I want the error to be similar to `Failed while trying to getting the app disabled`.

Reply:
Fixed by f302c7d